process/graduation_criteria.adoc

```diff
@@ -7,6 +7,8 @@ Projects of all maturities have access to all resources listed at https://cncf.i
 === Sandbox Stage
 
 To be accepted in the sandbox a project must have at least 2 TOC sponsors. See the https://github.com/cncf/toc/blob/master/process/sandbox.md[CNCF Sandbox Guidelines v1.0] for the detailed process.
+ 
+ * Adopt the CNCF https://github.com/cncf/foundation/blob/master/code-of-conduct.md[Code of Conduct].
```

Comment:
This should be added after "must have at least 2 TOC sponsors" before "See the ... for detailed process" of line 9. We probably should just break down line 9 like Incubation Stage and Graduation Stage doc style.